### ISSUE TITLE
Unable to die or continue progress

### ISSUE THREAD

**Issue by Vexereee** [DarkPacks/SkyFactory-4#1640]
## Bug Description

In 2 different survival worlds, I became unable to die through any means (mobs, fall damage, lava, self-harm with items such as Cyclic's Engraved Thunder, kill command etc), aggro any mobs whatsoever, access any inventory other than my own (including the one given by the inventory upgrade cake) and take damage at all. I am still able to use items as long as they don't have inventories (backpack, ender pouch etc) and harm mobs. I also tried using two items/blocks that had already been placed in the world (the parabox and the TelePastries nether cake), which proved to be successful. In the first world, it happened after I became an android through the mad scientist and turned back into a human with the blue pill afterwards. In the second world, I had already spent some time being an android (ate the red pill this time), after which I went to the Twilight Forest using the cake. Inside the Lich's castle is where the problem begun again. My HP icon turned red and instead of a percentage I was shown a question mark.   

### Did this behavior use to work in the previous version?

 I don't know. This is the first version I've played Skyfactory in.

## Steps to Reproduce (for bugs)

1. Become an android through either means.
2. Use the blue pill.
3. If step 2 didn't reveal a bug, in a different world or a backup of that same world, instead of turning back into a human, go to the twilight forest, inside the lich's castle. I'm clueless as to what could be found there though.

## Logs

* Client/Server Log: https://gist.github.com/Vexereee/478bf718ea75d9b1097080881a08494b
* Crash Log:

## World Information
* World Preset: Sky Factory 4 (World type: topography)
* Prestige Enabled: yes
* Modpack Version world created in: 4.2.2
* Additional Content Installed: no

## Client Information
* Modpack Version: 4.2.2 (the version I choose on the client: Release 1.12.2-forge-14.23.5.2854)
* Java Version: 8.0.2210.11 (update 221)
* Launcher Used:TLauncher
* Memory Allocated: 15 GB
* Server/LAN/Single Player: Single Player
* Optifine Installed: no
* Shaders Enabled: no

**Comment by PatLaPoint** [comment]
I have the same issue, no lives left and cannot die.
cannot interact with any items either

**Comment by PatLaPoint** [comment]
My solution was to edit the level.dat file and set Player ->  Health to like 5 or something.

**Comment by PatLaPoint** [comment]
i went back when i got hit

**Comment by Vexereee** [comment]
Thanks for your reply! Do you think your issue was also related to Matter Overdrive? Also, I may need some help with trying out your solution. My level.dat file opens a GOM Player. If I try to open it with Notepad, I just get gibberish. I'm kinda bad with stuff like this so assistance would be much appreciated :P

**Comment by PatLaPoint** [comment]
no, i've just had som cyclic hearts and was about to do some trials from deep mob, ended up restoring from aroma.
haven't had the issue since restoring.

**Comment by MOSTWANTEDGamer1** [comment]
Hello guys, I just got the exact same problem and don't know how to fix it. If you guys can, please help me out. Thank you and have a great day.

**Comment by MOSTWANTEDGamer1** [comment]
> no, i've just had som cyclic hearts and was about to do some trials from deep mob, ended up restoring from aroma.
> haven't had the issue since restoring.

It;s been a little while sine you had this problem, how did you fix it?
